xtable-core/src/test/java/org/apache/xtable/hudi/TestBaseFileUpdatesExtractor.java

```diff
@@ -135,7 +136,7 @@ void convertDiff() {
     expectedPartitionToReplacedFileIds.put(
         partitionPath1, Arrays.asList(fileName3, fileIdForFile4));
     expectedPartitionToReplacedFileIds.put(partitionPath2, Collections.singletonList(fileName5));
-    assertEquals(
+    assertEqualsIgnoreOrder(
         expectedPartitionToReplacedFileIds, replaceMetadata.getPartitionToReplacedFileIds());
```

Comment:
This can be simplified to ? 
`assertEquals(new HashSet<>(expectedPartitionToReplacedFileIds), new HashSet<>(replaceMetadata.getPartitionToReplacedFileIds())) `

Reply:
My bad didn't see it was `Map<String, List<String>>`, change LGTM. 

`Hudi does not depend on the order of the files in the lists?`
Yes it does not depend on order.